Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift

```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import SnapKit
+import UIKit
+
+struct TabTrayV2ControllerUX {
+    static let CornerRadius = CGFloat(4.0)
```

Comment:
Legacy code uses capitals on constants, but that doesn't match Swift style guidelines. These should start with a lower-case because it is new code.